eZ/Publish/Core/Persistence/Legacy/Tests/Content/MapperTest.php

```diff
@@ -637,4 +786,39 @@ static function ($languageCode) use ($languages) {
 
         return $this->languageHandler;
     }
+
+    /**
+     * @return \eZ\Publish\SPI\Persistence\Content\Type\Handler&\PHPUnit\Framework\MockObject\MockObject
+     */
+    protected function getContentTypeHandler()
```

Comment:
Missing strict type:

```suggestion
    protected function getContentTypeHandler(): Handler
```
(convention: we type-hint intersection for mocks via PHPDoc, but strictly type the actual type being mocked)